BinaryEncoding.md

```diff
@@ -500,6 +504,47 @@ where a `local_name` is encoded as:
 | index | `varuint32` | the index of the function whose locals are being named |
 | local_map | `name_map` | assignment of names to local indices |
 
+#### Label names
+
+The label names subsection assigns `name_map`s to a subset of functions in the
+[function index space](Modules.md#function-index-space). This may include both
+module-defined or imported functions, but is only meaningful for module-defined
+functions. The `name_map` for a given function assigns names to labels within
+the function, with labels identified by the offset from the start of the function's
```

Comment:
Hm, using raw byte offsets pierces the abstraction level on which the rest of the binary format is defined and makes the mapping a bit less robust. I wonder if there is an alternative. For example, couldn't we just assume an implicit numbering of a function's structured control instructions in order of appearance?